packages/demo/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@lucca-front/demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build-icons": "mkdirp ./dist/icons/font && cp -r ../icons/font/* ./dist/icons/font",
```

Comment:
build-icons just copies whereas build-scss does build and postbuild does the copy. it would be more coherent to have postbuild-icons do the copy and build-icons do nothing